### PR TITLE
Refactor sensor profile characterzation

### PR DESCRIPTION
**Pull request by vincentmigot**

### CODE REVIEW

File: views/sensor/characterize.php [hunk 1]
```diff
@@ -13,22 +13,435 @@
...
+                    validation = false;
+                }
+            }
+
+            return validation;
+        }
+    </script>
+
+    <?php 
+        function createValidatedInput($name, $label, $type, $unit = null) {
```
Comment (annetireau) [inline views/sensor/characterize.php]: could you add  a comment

File: views/sensor/characterize.php [hunk 2]
```diff
@@ -13,22 +13,435 @@
...
+                        addRequiredField("$name", "$label");
+                    })
+                </script>
+EOT;
+            $toReturn .= '</div>';
+            
+            return $toReturn;
+        }
+        
+        function createValidatedInputTableRow($name, $count, $label, $type, $unit = null, $extraText = "") {
```
Comment (annetireau) [inline views/sensor/characterize.php]: could you add  a comment

File: views/sensor/characterize.php [hunk 3]
```diff
@@ -13,22 +13,435 @@
...
+                </script>
+EOT;
+                        
+            $toReturn .= '</tr>';
+            
+            return $toReturn;
+        }
+    ?>
+    
+    <div class="characterize-form well" onSubmit="return validateRequiredFields();">
```
Comment (annetireau) [inline views/sensor/characterize.php]: could you add  a comment on this part

**Comment by vincentmigot** [comment]
Comments added
Fix bug on sensor caracterization numbers wich can be decimal sometimes
@see https://trello.com/c/ASb5BZiE/80-impossible-de-rentrer-des-nombres-%C3%A0-virgule-lorsquon-caract%C3%A9rise-un-lidar-seuls-des-nombres-entiers-peuvent-%C3%AAtre-entr%C3%A9s-ce-qui-d

**Pull request merged**